test/f90_correct/src/import_mod_from_user_dir1_m2.f90

```diff
@@ -0,0 +1,12 @@
+module import_source
```

Comment:
Missing standard copyright template, and a comment about what the file is part of.

Reply:
Oh, I missed that. Fixed. Thanks.